test/core/layout/layout.test.js

```diff
@@ -15,6 +15,7 @@ jest.mock("../../../src/core/layout/gel-group.js");
 
 describe("Layout", () => {
 	const sixGelButtons = ["achievements", "exit", "howToPlay", "play", "audio", "settings"];
+	const expectedNumberOfGroupLayouts = 12;
```

Comment:
`expectedLayouts`

Added a note that we should discuss naming variables at the next dev meet - our approach is to try and be concise where possible.

Also could you do me a favour and rename `sixGelButtons` to `gelButtons` as that's an awful name that it appears I added 😂